app/views/invoice/pdf.haml

```diff
@@ -207,7 +207,7 @@
               %dt= t(:phone)
               %dd= @invoice.buyer_phone
 
-            - if @invoice.buyer_phone.present?
+            - if @invoice.buyer_url.present?
```

Comment:
I think it makes sense to get rid of this check completely. Neither `admin` or `registrar` area invoice views have it. https://github.com/internetee/registry/blob/acab58b71cf367754e378489fc1d8a42724e6941/app/views/registrar/invoices/partials/_buyer.haml#L20
@vohmar is OK with this.

Reply:
I also had an idea to rename it to `buyer_website` to be more generic, but this could be done later.

Reply:
You are talking about check but you are not referencing one, you want me to remove the if statements from the haml ? I think this will end with nil errors

Reply:
check = `if` statement on line no. 210.

Reply:
As you wish, removed the if's.